content/blog/2021/04/2021-04-07-contributhon-participants.adoc

```diff
@@ -0,0 +1,105 @@
+
+== Sharon Jebitok
+
+image:/images/post-images/2020-04-contributhon-participants/sharon-jebitok.jpg[Sharon Jebitok, width=250px]
+
+Sharon is based in Kenya.
+She's a software development student at Microverse, a remote school for software developers.
+She has been an active member of She Code Africa and she's presently a mentee on She Code Africa's Contributhon Program with the Jenkins project.
+She has spent one year in the software industry since she switched her career into tech, learning to write meaningful code and collaborate with other developers/designers and the tech community.
+
+You'll recognize her contributions as link:https://github.com/jebitok-dev[`jebitok-dev`] on GitHub and as link:https://issues.jenkins.io/issues/?jql=creator%20%3D%20%20jebitok16_[`jebitok16_`] on the link:https://issues.jenkins.io/issues/?jql=creator%20%3D%20%20jebitok16_[Jenkins issue tracker].
+
+== Esther Ejidike
+
+image:/images/post-images/2020-04-contributhon-participants/esther-ejidike.jpg[Esther Ejidike, width=250px]
+
+Esther is a frontend developer based in Lagos, Nigeria.
+She is one of the participants in the She Code Africa Contributhon Open Source Program working with Jenkins and will be contributing to the nodes and processes plugin.
+She loves to convert designs to exact replicas in the form of webpages and she likes to learn new things in her free time.
+
+You'll recognize her contributions as link:https://github.com/Esther101[`Esther101`] on GitHub and as link:https://issues.jenkins.io/issues/?jql=creator%20%3D%20%20esther101[`esther101`] on the link:https://issues.jenkins.io/issues/?jql=creator%20%3D%20%20esther101[Jenkins issue tracker].
+
+== Cynthia Iradukunda
+
+image:/images/post-images/2020-04-contributhon-participants/cynthia-iradukunda.png[Cynthia Iradukunda, width=250px]
+
+Cynthia Iradukunda is based in Kigali, Rwanda.
+She is currently a junior Computing student at the African Leadership University (ALU) in Mauritius.
+She wants to be a software engineer because it will help her solve real-world problems while also allowing her to use her coding skills.
+Her coding skills include but not limited to Java, JavaScript, and attention to detail.
+By contributing to the Git plugin, she hopes to help users have a smooth process using its documentation, get involved with the project, and connect with other community members.
+
+You'll recognize her contributions as link:https://github.com/ciradu2204[`ciradu2204`] on GitHub and as link:https://issues.jenkins.io/issues/?jql=creator%20%3D%20%20ciradu2204[`ciradu2204`] on the link:https://issues.jenkins.io/issues/?jql=creator%20%3D%20%20ciradu2204[Jenkins issue tracker].
+
+== Lucy Karimi
+
+// image:/images/post-images/2020-04-contributhon-participants/lucy-karimi.jpg[Lucy Karimi, width=250px]
```

Comment:
https://avatars.githubusercontent.com/u/68492502?v=4 could be rotated/used, but I am not sure. Maybe there was a preference to publish without an image

Reply:
Lucy has connected on Slack and says that she'll provide a bio and a picture.  I'll revise the post in a new pull request with her update once I receive it.